src/main/java/practice/StreamPractice.java

```diff
@@ -1,11 +1,18 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private static final int EVEN_NUMBERS_INDEX = 2;
```

Comment:
```suggestion
    private static final int EVEN_NUMBERS_INDEX = 2;
```
maybe better delete this constant and divide in checks with 2 or rename this constant but it`s only my opinion you can do this or no